example_configs/wildfly-10.yaml

```diff
@@ -25,3 +25,15 @@ rules:
       server: $1
       http_listener: $2
 
+  - pattern: "^org.xnio<type=Xnio, provider=\"nio\", worker=(.+)><>(.+_size|.+_count):"
+    attrNameSnakeCase: true
+    name: wildfly_xnio_$2
```

Comment:
Is xnio actually part of wildfly? It seems to be a separate project.

Reply:
Xnio is a jboss project that ships as part of undertow. Undertow does not expose all metrics about worker threads but leaves that to xnio.

Reply:
If it's not part of the wildfly component, it doesn't get a wildfly prefix as it is its own thing.